dangerfile.js

```diff
@@ -1,73 +0,0 @@
-const PR_NUMBER = danger.github.pr.number;
-const PR_URL = danger.github.pr.html_url;
-const PR_LINK = `[#${PR_NUMBER}](${PR_URL})`;
-
-function getCleanTitle() {
-  let title = danger.github.pr.title;
-  // remove fix(component): prefix
-  title = title.split(": ").slice(-1)[0].trim();
-  // remove links to JIRA tickets, i.e. a suffix like [ISSUE-123]
-  title = title.split("[")[0].trim();
-  // remove trailing dots
-  title = title.replace(/\.+$/, "");
-
-  return title;
-}
-
-function getChangelogDetails() {
```

Comment:
Please bring back these instructions and write them to a [job summary](https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/). Without such a description, it's going to be harder to get it right.